cardano/src/block/types.rs

```diff
@@ -223,7 +223,7 @@ impl SscProof {
     }
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Why are these impls gone now, they were good?

Reply:
It just that I have added them to implement the specific code, and then code has changed, so I didn't want to introduce new stuff, without using it right away